letsencrypt/client/network2.py

```diff
@@ -280,26 +321,19 @@ def answer_challenge(self, challb, response):
         try:
             authzr_uri = response.links['up']['url']
         except KeyError:
-            raise errors.NetworkError('"up" Link header missing')
+            # TODO: Right now Boulder responds with the authorization resource
+            # instead of a challenge resource... this can be uncommented
+            # once the error is fixed.
+            return None
+            # raise errors.NetworkError('"up" Link header missing')
         challr = messages2.ChallengeResource(
             authzr_uri=authzr_uri,
             body=messages2.ChallengeBody.from_json(response.json()))
         # TODO: check that challr.uri == response.headers['Location']?
         if challr.uri != challb.uri:
-            raise errors.UnexpectedUpdate(challr.uri)
+            raise errors.UnexpectedUpdate(challb.uri)
```

Comment:
I copied my comment above to https://github.com/letsencrypt/acme-spec/issues/110